src/arrays/programs_involving_multidimensional_data/easy/q4/Rotation.java

```diff
@@ -0,0 +1,33 @@
+package arrays.programs_involving_multidimensional_data.easy.q4;
+
+/*
+Write a Java program that initializes a 2D array of random integers of size 4 by 4, with the range of values from 1 - 99
```

Comment:
Change to:
Make a Java program that initalizes a 4 by 4 2D array of random integers ranging between 1 and 99 inclusive of both values. Then, "rotate" the elements in the four corners of the array clockwise by passing the array to a method called rotateArray. The method should be able to handle a square 2D array of any size. Print the array before the rotation in the main method, and the array after the rotation (including the line declaring it to be after the rotation) in rotateArray. 

Rotation example:
Before the rotation:
17 ...  30 
...        ...
47 ... 90 
After the rotation:
47 ...  17 
...        ...
90 ... 30